website/thaliedje/templates/thaliedje/search.html

```diff
@@ -4,11 +4,14 @@
 <div id="player-search-container-{{ player.id }}">
     <input v-model="query" id="search" type="text" maxlength="100" placeholder="Search for something to play" class="container-fluid form-control mb-3"/>
     <div id="results" class="px-3 search-list">
-        <div class="d-flex flex-column">
-            <template v-if="tracks.length > 0 && query !== ''">
+        <div class="row">
+            <div class="col-lg-4" v-if="tracks.length > 0 && query !== ''">
                 <h3>Tracks</h3>
                 <template v-for="(track, index) in tracks">
                     <div class="d-flex my-2">
+                        <div class="me-1">
+                            <img height="50" v-if="track.image !== null" :src="track.image"/>
+                        </div>
                         <div class="flex-equal-width"><% track.name %></div>
```

Comment:
Maybe even better would be to have 1 (wider) column with the track name and on the next line the artists, instead of having 2 columns.